docs/visualization/wcsaxes/overlaying_coordinate_systems.rst

```diff
@@ -99,3 +99,44 @@ a tuple to other methods.
     overlay['dec'].set_ticklabel(color='magenta', size=6)
     overlay['dec'].set_axislabel_position('t')
     overlay['dec'].set_axislabel('Declination', color='magenta')
+
+Overlaying a Compass Arrow
+**************************
+
+It is often useful to add compass arrows to your images, denoting which directions correspond to North and East on the sky.
+The function :meth:`~astropy.wcs.utils.north_pole_angle` calculates the correct angle for this compass, which can easily be displayed using a matplotlib :class:`~matplotlib.mpl_toolkits.axes_grid1.anchored_artistsAnchoredDirectionArrows()` artist.
+
+
+.. plot::
```

Comment:
Hard for me to tell what this is supposed to show me unless the RTD build is fixed. Would be nice to actually overlay this on `imshow` since I think that is how this compass is meant to be used?